Apps/W1/Shopify/app/src/GraphQL/Codeunits/ShpfyGQLSalesChannels.Codeunit.al

```diff
@@ -0,0 +1,27 @@
+namespace Microsoft.Integration.Shopify;
+
+/// <summary>
+/// Codeunit Shpfy GQL SalesChannels (ID 30371) implements Interface Shpfy IGraphQL.
+/// </summary>
+codeunit 30371 "Shpfy GQL SalesChannels" implements "Shpfy IGraphQL"
+{
+    Access = Internal;
+
+    /// <summary>
+    /// GetGraphQL.
+    /// </summary>
+    /// <returns>Return value of type Text.</returns>
+    internal procedure GetGraphQL(): Text
+    begin
+        exit('{"query": "{publications(first: 10, catalogType: APP) { edges { node { id catalog { id ... on AppCatalog { apps(first: 1) { edges { node { id handle title } } } } } } } } }"}');
```

Comment:
Only 10? No paging?

Reply:
10 might be too little. Maybe 50 or paging

Reply:
changed it to use paging